README.md

```diff
@@ -7,11 +7,26 @@ A language server implementation based on the [Language Server Protocol](https:/
 
 This project contains:
 
-- [MicroProfile JDT LS Extensions](./microprofile.jdt)
 - [MicroProfile Language Server](./microprofile.ls)
+- [MicroProfile JDT LS Extensions](./microprofile.jdt)
 
 You can build all projects at once by running the `buildAll.sh` script (`buildAll.bat` on Windows).
 
+The language features provided by the MicroProfile language server depend on the user's Java project.
+For example, MicroProfile Fault Tolerance features (snippets, validation, completion, etc.) are only provided if the user's Java project has
+MicroProfile Fault Tolerance as a depenency.
+
+In order for the MicroProfile Language Server to provide relevant features within a given Java project, the language server relies on an external
+complement such as MicroProfile JDT (see [MicroProfile JDT LS Extensions](./microprofile.jdt)) to scan/inspect the Java project.

```

Comment:
Maybe we could provide some more information on what the external component / microprofile jdt is responsible for. We don't have much information listed in the microprofile.jdt README here either https://github.com/eclipse/lsp4mp/tree/master/microprofile.jdt

Reply:
Fixed, not sure if this is good enough or if there's something I'm missing:

![image](https://user-images.githubusercontent.com/20326645/89330785-c0509d80-d65e-11ea-9fcc-fdd55b16e828.png)


> This external component is used to help resolve LSP commands sent from the MicroProfile Language Server.
See [`MicroProfileLanguageClientAPI.java`](https://github.com/eclipse/lsp4mp/blob/master/microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/ls/api/MicroProfileLanguageClientAPI.java).

> For example, when the MicroProfile Language Server calls [`MicroProfileProjectInfoProvider#getProjectInfo`](https://github.com/eclipse/lsp4mp/blob/82c4798ac142efeb06499ece053407ef01009d46/microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/ls/api/MicroProfileProjectInfoProvider.java#L35),
the `microprofile/projectInfo` command is handled in the external component with 
[`MicroProfileDelegateCommandHandler#executeCommand`](https://github.com/eclipse/lsp4mp/blob/82c4798ac142efeb06499ece053407ef01009d46/microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/internal/core/ls/MicroProfileDelegateCommandHandler.java#L53).

Reply:
I think it is good. unless maybe @angelozerr has some ideas of any other important information about the microprofile.jdt that should be included.